Framework/Kernel/src/TimeSeriesProperty.cpp

```diff
@@ -801,6 +800,11 @@ template <typename TYPE> double TimeSeriesProperty<TYPE>::timeAverageValue(const
     if ((timeRoi == nullptr) || (timeRoi->useAll())) {
       const auto &filter = getSplittingIntervals();
       retVal = this->averageValueInFilter(filter);
+    } else if (timeRoi->useNone()) {
+      // if TimeROI bans everything, use the simple mean
```

Comment:
why this option? a `NaN` seems more appropriate to me :thinking:

Reply:
This is, unfortunately, the old behavior